CMakeLists.txt

```diff
@@ -245,9 +248,13 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/third_party/libosmium/cmake")
 set(OSMIUM_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/libosmium/include")
-find_package(Osmium REQUIRED COMPONENTS io)
-add_dependency_includes(${OSMIUM_INCLUDE_DIR})
-
+if(NOT ROUTED_STANDALONE)
+  find_package(Osmium REQUIRED COMPONENTS io)
+  add_dependency_includes(${OSMIUM_INCLUDE_DIR})
+else()
+  # transitive dependency
+  include_directories(SYSTEM ${OSMIUM_INCLUDE_DIR})
```

Comment:
why do you have a switch here? Adding OSMIUM include path to the search path while not calling find_package will be a no-op.